versioned_docs/version-latest/learn/monitoring.md

```diff
@@ -0,0 +1,723 @@
+
+---
+
+### UFW Configuration
+
+On the Prometheus Server:
+```bash
+sudo ufw allow from <farmer-server-1-ip> to any proto tcp port 9100 comment 'Exporter-farmer-1'
+sudo ufw allow from <farmer-server-2-ip> to any proto tcp port 9100 comment 'Exporter-farmer-2' 
+```
+Repeat this configuration for other farmer servers as necessary.
+
+On the Monitored Servers:
+```bash
+sudo ufw allow from <prometheus-server-ip> to any proto tcp port 9100 comment 'Exporter-PrometheusServer'
+```
+
+### Service Management Commands:
+
+```bash
+sudo systemctl start prometheus			
+# Start Prometheus service
+
+sudo systemctl status prometheus			
+# Check Prometheus service status
+
+sudo systemctl enable prometheus			
+# Enable Prometheus service for autostart
+
+sudo systemctl disable prometheus			
+# Disable autostart for Prometheus service
+```
+
+---
+
+Here's a revised version for improved readability:
+
+---
+
+## 3. Grafana Server
+
+You have the option to utilize Grafana Cloud instead of managing and scaling your instance of Grafana. [Sign up for a free account](https://grafana.com/auth/sign-up?pg=download&plcmt=box-right), offering free access to 10k metrics, 50GB logs, 50GB traces, and more.
+
+### Installation Guide - Latest Open Source Grafana Release (APT repository)
+
+:::note
+Choose between:
+- Enterprise: Identical to the open-source version but offers additional features unlockable with a license.
+- Open Source: Functionally identical to the Enterprise version without enterprise-specific features.
+:::
+
+### Steps to Download
+
+```bash
+sudo apt-get install -y apt-transport-https software-properties-common wget
+sudo mkdir -p /etc/apt/keyrings/
+wget -q -O - https://apt.grafana.com/gpg.key | gpg --dearmor | sudo tee /etc/apt/keyrings/grafana.gpg > /dev/null
+echo "deb [signed-by=/etc/apt/keyrings/grafana.gpg] https://apt.grafana.com stable main" | sudo tee -a /etc/apt/sources.list.d/grafana.list
+sudo apt-get update
+sudo apt-get install grafana
+# Alternatively, use `sudo apt-get install grafana-enterprise` for the Enterprise version.
+```
+
+> *These commands execute a post-install script*:
+```bash
+addgroup --system grafana
+adduser --system --home /usr/share/grafana --no-create-home --ingroup grafana --disabled-password --shell /bin/false grafana
+```
+> They grant necessary permissions for the system user 'grafana' to required folders.
+
+:::note
+In some cases during installation, you might encounter the error:  
+> cannot access /etc/grafana/*: No such file or directory.  
+
+This might occur due to VM permissions issues in creating missing configuration files. A solution is to navigate to the `/var/cache/apt/archives` directory—where apt stores downloaded packages before installation—or manually download the `.deb` file and use `dpkg --force flag`. For example: `dpkg --force-all -i grafana_10.2.2_amd64.deb`
+:::
+
+---
+
+### Starting Server
+
+```bash 
+sudo systemctl daemon-reload
+```
+```bash
```

Comment:
No service `enable`? Will the service auto start after a reboot without it?

Reply:
Added service management commands to the Grafana section!